eoscostarica/eos-rate
# Fix/293 remove bp status

### GH Filter active Block Producer

#293 

### Steps to test

1. go to EOS RATE
1. click and see all blocks producers list
1. click one block producer and see profile (status label should be removed)

### Checklist
